### ISSUE TITLE
Show where your uploaded images are used 

### ISSUE THREAD

**Issue by joerundp** [commons-app/apps-android-commons#1958]
I would like to be able to see what images contributed to Wikimedia are used, and also the pages they are used at. It already lists the amount of contributions used, but not which ones and on what pages. This would be very helpful.

**Comment by nicolas-raoul** [comment]
A link to a web page that has all of this information is probably the best way to implement this.
What page do you think would be best? I am sure I saw such a page some time ago.

**Comment by joerundp** [comment]
Well yes, that could work. Some images are used on several pages though, so
perhaps the image used on ie. Wikipedia English (if present) followed by
'also used on...' below. Then next image. Also the ability to sort by time
of upload, so it can show the most recent uploads. I think, as a
contributor, the 'reward' of seeing your work being used should have some
significance.

Den man. 29. okt. 2018, 09:04 skrev Nicolas Raoul <notifications@github.com

**Comment by chrisdebian** [comment]
I think this would really encourage users, and a link would be a good addition to anything received in Notifications.

Maybe at the top of the "Contributions" tab, have a URL, with the text "See how your contributions are being used"

Chris

**Comment by parneet-guraya** [comment]
I guess this #6006 solves this issue.

**Comment by nicolas-raoul** [comment]
@parneet-guraya Yes this shows usage for a specific file, but I guess here the goal is to see all usages without having to tap each of your uploaded pictures.

Given our limited resources, we should focus on areas where a native app has clear advantages over a website, this is not such an area.
So I would recommend to:
1. Add a [link to GLAMorous](https://glamtools.toolforge.org/glamorous.php?doit=1&username=Syced&use_globalusage=1&ns0=1&show_details=1&projects[wikipedia]=1&projects[wikimedia]=1&projects[wikisource]=1&projects[wikibooks]=1&projects[wikiquote]=1&projects[wiktionary]=1&projects[wikinews]=1&projects[wikivoyage]=1&projects[wikispecies]=1&projects[mediawiki]=1&projects[wikidata]=1&projects[wikiversity]=1) called `See where my images are used` to the tooltip of `Images used` in Achievements.
2. Send a Feature Request to GLAMorous asking to make it more mobile-friendly.
